db-migrations/11_update_merlin_app_config.down.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE applications
+SET config = '{}'
```

Comment:
previously there is no sections?

Reply:
Yup. `config` column is a new additional column to application table. It was introduced by Turing (https://github.com/gojek/mlp/blob/main/db-migrations/7_application_config.up.sql).